content/docs/04.5-devx/07-sandbox-clusters.md

```diff
@@ -47,3 +47,57 @@ To create your new sandbox cluster:
    * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
 
 <br />
+
+
+## Palette Virtual Cluster Pause and Resume
+
+The Palette allows the Pause and Resume of Palette Virtual Clusters when not in use. This feature enables the users with optimized resource sharing by pausing the virtual clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette virtual clusters. 
+
+<br />
+
+### Resource Dependencies
```

Comment:
This title is confusing. The content below sounds more like prerequisites, but it's also mixing general information. The two need to be separated.